andrada12/assignment3/Dictionary.java

```diff
@@ -0,0 +1,62 @@
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ *
+ * @author andrada
+ */
+public class Dictionary {
+    
+    static HashMap<String, ArrayList<String>> dictionary = new HashMap<String, ArrayList<String>>();
+    
+    public static void setDictionary(String[] words){
+        
+        for(int i = 0; i < words.length; i++){
```

Comment:
FYI: Did you know you could do for(String word : words) {
for the outer loop? This is optional, but many people find it more readable.